src/muse/investments.py

```diff
@@ -186,19 +187,13 @@ def cliff_retirement_profile(
     Assets with a technical life smaller than the input time-period should automatically
     be renewed.
 
-    Hence, if ``technical_life <= protected``, then effectively, the technical life is
-    rewritten as ``technical_life * n`` with ``n = int(protected // technical_life) +
-    1``.
-
     We could just return an array where each year is represented. Instead, to save
     memory, we return a compact view of the same where years where no change happens are
     removed.
 
     Arguments:
         technical_life: lifetimes for each technology
-        current_year: current year
-        protected: The technologies are assumed to be renewed between years
-            `current_year` and `current_year + protected`
+        investment_year: The year in which the investment is made
```

Comment:
The year index? It is not the actual year, as in 2025. And if it is, it should not default to zero - if defaulting to anything.